python/year-service/yearservice/yearservice/settings.py

```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['127.0.0.1', 'localhost', 'year']
```

Comment:
Does 127.0.0.1 vs 0.0.0.0 matter here?

Reply:
I don't think so. When running on localhost, `127.0.0.1` and `locahost` are legitimate domains for this service. When running behind a load-balancer for all the Year services, `year` is the legitimate domain for the service.